cmd/deploy/deploy.go

```diff
@@ -332,12 +332,14 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 	}
 
 	err = deployer.deploy(ctx, deployOptions)
+	if _, ok := deployer.(*remoteDeployCommand); ok {
```

Comment:
we need to check if exec is remote or not. If deploy has been executed in remote means the following logic has been already executed in remote machine so we will set err = nil to not log twice command error

Reply:
If we return it as no error how is the main pipeline knowing that the remote deployment is failing?

Reply:
being main pipeline, why I would like to know that is failing? shouldn't the remote handle its own errors?

Reply:
need to check it, draft status in the meantime